.github/workflows/appengine-cleanup.yml

```diff
@@ -0,0 +1,29 @@
+name: Automated AppEngine cleanup script
+on:
+  # Run once every week on Sunday
+  schedule:
+    - cron: "0 0 * * 0"
+  # To manually trigger workflow
+  workflow_dispatch: {}
+jobs:
+  appengine_cleanup:
+    runs-on: ubuntu-latest
+    env:
+      CI_OPTIONS: '--quiet'
```

Comment:
What does quiet do?

Reply:
Skips the `yes/no` requirement: https://cloud.google.com/sdk/gcloud/reference#--quiet

Reply:
Alternatively, it looks like you can do this without modifying the script by setting `CLOUDSDK_CORE_DISABLE_PROMPTS` to 1.

https://cloud.google.com/sdk/gcloud/reference/config/set#disable_prompts

Reply:
This seems like a better choice. I'll switch to this env variable.